src/chocolatey.tests.integration/context/dependenciesbeforemodify/hasdependencywithbeforemodify/1.0.0/hasdependencywithbeforemodify.nuspec

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>hasdependencywithbeforemodify</id>
+        <version>1.0.0</version>
+        <title>hasdependencywithbeforemodify</title>
+        <authors>__REPLACE_AUTHORS_OF_SOFTWARE__</authors>
+        <owners>__REPLACE_YOUR_NAME__</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>__REPLACE__</description>
+        <summary>__REPLACE__</summary>
+        <releaseNotes />
+        <copyright />
+        <tags>hasdependencywithbeforemodify admin</tags>
+        <dependencies>
+            <dependency id="isdependencywithbeforemodify" version="[1.0.0, 2.0.0.0)" />
```

Comment:
Was the four part version number here on purpose?

Reply:
I was copying existing packages, so yes, but I don't know why they were originally written this way. (see: `hasdependency`, for example)